include/grpcpp/impl/codegen/byte_buffer.h

```diff
@@ -98,6 +100,7 @@ class ByteBuffer final {
 
  private:
   friend class SerializationTraits<ByteBuffer, void>;
+  friend class ServerInterface;
```

Comment:
I believe that this line is not an API change since ServerInterface is a class fully-defined internally and friendship is not inherited. No objections to that line.